test/jdk/javax/swing/JTableHeader/SerializeJTableHeader.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 6450193
+ * @summary JTableHeader doesn't uninstall UI after first serialization
+ * @run main SerializeJTableHeader
+ */
+
+import javax.swing.table.JTableHeader;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+import javax.swing.SwingUtilities;
+
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectOutputStream;
+
+public class SerializeJTableHeader implements Runnable{
+
+    public void run() {
+        JTableHeader jth = new JTableHeader();
+        try {
+            for(int i = 0; i < 10; i ++){
+                ByteArrayOutputStream baos = new ByteArrayOutputStream();
+                ObjectOutputStream oos = new ObjectOutputStream(baos);
+                //System.out.println(jth.getUI());
+                oos.writeObject(jth);
+                oos.writeObject(jth.getAccessibleContext());
+            }
+        }
+        catch (Exception x) {
+            x.printStackTrace();
+        }
```

Comment:
Shouldn't the test fail if an exception is thrown? Without the fix, `NotSerializableException` is expected, and the test will fail. But the test will never fail if you catch the exception to print a message.

```suggestion
        } catch (Exception x) {
            x.printStackTrace();
        }
```